polaris-react/config/rollup/plugin-styles.js

```diff
@@ -148,16 +148,9 @@ module.exports.styles = function styles({
 
       let sassOutput;
       try {
-        sassOutput = nodeSass
-          .renderSync({
-            data: source,
-            file: id,
-            outputStyle: 'compact',
-            includePaths: [path.dirname(id)],
-          })
-          .css.toString();
+        sassOutput = (await sass.compileAsync(id, {style: 'compressed'})).css;
```

Comment:
https://sass-lang.com/blog/new-js-api-release-candidate#how-to-use-it

> As with the old API, the synchronous functions are substantially faster than their async counterparts, so we recommend using them if at all possible.

Should we stick to using `sass.compile`?

Reply:
That doesn't appear to be the case with `sass-embedded`:
> [Unlike the sass package, the asynchronous API in sass-embedded will generally be faster than the synchronous API since the Sass compilation logic is happening in a different process. ](https://www.npmjs.com/package/sass-embedded#:~:text=Unlike%20the%20sass%20package%2C%20the%20asynchronous%20API%20in%20sass%2Dembedded%20will%20generally%20be%20faster%20than%20the%20synchronous%20API%20since%20the%20Sass%20compilation%20logic%20is%20happening%20in%20a%20different%20process.)